src/partials/options.html

```diff
@@ -1,327 +1,317 @@
-<div class="editor-row options-tab">
-  <div class="gf-form">
-    <gf-form-switch
-      class="gf-form"
-      label="Multibar"
-      label-class="query-keyword width-8"
-      checked="ctrl.isMultibar"
-    />
-  </div>
-
-  <br/>
-  
-  <div class="section gf-form-group" ng-if="ctrl.isMultibar">
-    <div class="gf-form">
-      <label class="gf-form-label width-8">Key column</label>
-      <div class="gf-form-select-wrapper width-14">
-        <select
-          class="gf-form-input"
-          ng-model="ctrl.panel.keyColumn"
-          ng-options="n for n in ctrl.columns"
-          ng-change="ctrl.render()"
-        />
+<!DOCTYPE html>
+<html>
```

Comment:
why do you wrap it in `<html>`?

Reply:
it was I mistake, I tried to use https://validator.w3.org/ and that's I wrapped the file. It was required only for testing.